modules/yunhei/user.py

```
@@ -94,30 +95,35 @@ async def check(msg: Bot.MessageSession, qqnum: str = "all"):
+                            # 严重的特殊反应
+                            if user_info['level'] == "严重":
+                                account = user_info['account_name']
+                                try:
+                                    await msg.call_api("set_group_kick",
+                                                       group_id=str(msg.target.target_id).split('|')[-1],
+                                                       user_id=int(account), reject_add_request=True)
+                                except Exception as e:
+                                    await msg.finish(f"踢出用户失败：{e}")
+                                name = await get_qqname(msg, account)
+                                severe_summary.append(
+                                    f"{name}（{account}）\n违规原因：{
```

Pylint (3.3.7)

[error] Parsing failed: 'unterminated string literal (detected at line 122) (modules.yunhei.user, line 122)'

(E0001)